apps/api/src/domain/models/audio-visual/video/repositories/arango-video-query-repository.spec.ts

```diff
@@ -149,6 +154,62 @@ describe(`ArangoVideoQueryRepository`, () => {
         });
     });
 
+    describe(`tag`, () => {
+        const existingTagLabel = 'plants';
+
+        const existingTag = buildTestInstance(EventSourcedTagViewModel, {
+            id: buildDummyUuid(90),
```

Comment:
Do we need all of these props? If so, we might want a test setup helper for all tag test cases for queries.